lib/node_modules/@stdlib/stats/iter/README.md

```diff
@@ -103,11 +103,57 @@ var o = ns;
 
 <!-- eslint no-undef: "error" -->
 
+<!-- Here are a few examples showcasing the usage of functions from the @stdlib/stats/iter namespace: -->
+
+
 ```javascript
-var getKeys = require( '@stdlib/utils/keys' );
-var ns = require( '@stdlib/stats/iter' );
 
-console.log( getKeys( ns ) );
+// Computing Cumulative Sum
+
+
+var ns = require('@stdlib/stats/iter');
```

Comment:
Please follow stdlib style and format conventions.

Reply:
May u please help me in figuring out the correct style and format conventions

Reply:
I suggest studying the existing codebase and attempting to mimic what you see. E.g., spacing, quote style, etc.